app/src/main/java/com/amaze/filemanager/crashreport/ErrorActivity.java

```diff
@@ -271,6 +276,38 @@ public boolean onOptionsItemSelected(final MenuItem item) {
     return false;
   }
 
+  @Override
+  public void onConfigurationChanged(@NonNull Configuration newConfig) {
```

Comment:
These methods are duplicated a few times. Try to minimize code duplications, maybe a class hierarchy?

Reply:
I think we had a ThemedActivity to deal with themes, but I am unsure how adaptable that class is for this.

Reply:
> I think we had a ThemedActivity to deal with themes, but I am unsure how adaptable that class is for this.

I've moved 'em to `PreferenceActivity`...

Reply:
> `PreferenceActivity`

Why? Isn't that related to configurations?

Reply:
> Why?

`AboutActivity` extends `PreferenceActivity` directly (which is extended by `ThemedActivity` too), so I chose `PreferenceActivity`.

But, moving to `ThemedActivity` would work too, by updating `AboutActivity`, will do :)

Reply:
✅ Done